frontend/src/lib/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -387,6 +413,11 @@ export function PropertiesTable({
                 key: 'value',
                 title: 'Value',
                 render: function Value(_, item: any): JSX.Element {
+                    const isComplexStructure = Array.isArray(item[1]) || (isObject(item[1]) && item[1] !== null)
+
+                    if (isComplexStructure && featureFlags[FEATURE_FLAGS.CRM_ITERATION_ONE]) {
```

Comment:
This feels like something that doesn't need to be associated with your feature flag but does make sense to roll out on a flag to test a bunch of use cases on prod.